<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewUser</class>
 <widget class="QDialog" name="NewUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Create a new User</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>91</y>
     <width>621</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="Labels">
      <item>
       <widget class="QLabel" name="firstNameLabel">
        <property name="text">
         <string>First Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lastNameLabel">
        <property name="text">
         <string>Last Name: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="userNameLabel">
        <property name="text">
         <string>Username: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Password: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="reneterPasswordLabel">
        <property name="text">
         <string>Renter Password</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="firstName">
            <item>
             <widget class="QLineEdit" name="firstNameInput"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="lastName">
            <item>
             <widget class="QLineEdit" name="lastNameInput"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="userName">
            <item>
             <widget class="QLineEdit" name="userNameInput"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="passwordInput"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="renterPassword">
            <item>
             <widget class="QLineEdit" name="renterPasswordInput"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>460</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>460</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstNameRequired">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>110</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Required</string>
   </property>
  </widget>
  <widget class="QLabel" name="lastNameRequired">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Required</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordRequired">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>310</y>
     <width>126</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Required</string>
   </property>
  </widget>
  <widget class="QLabel" name="userNameRequired">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>240</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Username already taken</string>
   </property>
  </widget>
  <widget class="QLabel" name="renterPasswordRequired">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>380</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Please reneter password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
